### ISSUE TITLE
Feature: (sort-classes) Distinguish between normal methods and getters and setters

### ISSUE THREAD

**Issue by ElPrudi** [azat-io/eslint-plugin-perfectionist#45]
### What rule do you want to change?

sort-classes

### Describe the problem

I like the idea of this rule to sort all of my methods, but, at least with the natural config, `get` and `set` methods are buried between them. It's much more readable to put all of them in an extra group.

### Code example

This:
```ts
class A {
    x() {}
    b() {}
    get z() {}
    get c() {}
    set c() {}
}
```

Turns into:
```ts
class A {
    b() {}
    get c() {}
    set c() {}
    x() {}
    get z() {}
}
```

But I really prefer instead this:

```ts
class A {
    b() {}
    x() {}
    get c() {}
    set c() {}
    get z() {}
}
```

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Thanks for the issue. That's a good idea, I'll be sure to add it soon. 🙂

**Comment by azat-io** [comment]
Added in 73132e8091b18240d2a74aa8200e88d1547bec64.

Release is coming soon.

**Comment by azat-io** [comment]
Released in [v2.0.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.0)
